From 418fbadf0d30b07bc8fbe05fa379820b06d88b60 Mon Sep 17 00:00:00 2001
From: Stefan Fabian <fabian@sim.tu-darmstadt.de>
Date: Fri, 27 Mar 2020 10:16:10 +0100
Subject: [PATCH] Replaced private package name with public one.

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 376427b..13789fc 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,7 @@ Source: ros-melodic-ros-babel-fish
 Section: misc
 Priority: optional
 Maintainer: Stefan Fabian <ros_babel_fish@stefanfabian.me>
-Build-Depends: debhelper (>= 9.0.0), hector-melodic-ros-babel-fish-test-msgs, openssl, ros-melodic-actionlib, ros-melodic-catkin, ros-melodic-rosapi, ros-melodic-roscpp, ros-melodic-roscpp-tutorials, ros-melodic-roslib, ros-melodic-std-msgs
+Build-Depends: debhelper (>= 9.0.0), ros-melodic-ros-babel-fish-test-msgs, openssl, ros-melodic-actionlib, ros-melodic-catkin, ros-melodic-rosapi, ros-melodic-roscpp, ros-melodic-roscpp-tutorials, ros-melodic-roslib, ros-melodic-std-msgs
 Homepage: 
 Standards-Version: 3.9.2
 
-- 
2.17.1

